code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -78,8 +81,8 @@
 	flags = FPRINT
 	body_parts_covered = HIDEHAIR | MOUTH
 	w_class = W_CLASS_SMALL
-	species_fit = list(VOX_SHAPED, GREY_SHAPED)
-	species_restricted = list("exclude", INSECT_SHAPED)
+	species_fit = list(VOX_SHAPED, GREY_SHAPED, INSECT_SHAPED)
+	// species_restricted = list("exclude", INSECT_SHAPED)
```

Comment:
delete this instead of commenting it out

Reply:
Done